### DISCUSSION TITLE
Show release year for naming pattern

### DISCUSSION THREAD

**Discussion by nydave69** [pymedusa/Medusa#8422] [Ideas & Feature requests]
**Describe the bug**
There are so many duplicate shows now that I have them on disk as ShowName (Year)
The renamer has no ShowYear so it renames it as ShowName then I have to go in and rename it properly.
Some times I catch it early and sometime I don't...   Most of the time I don't and that causes the show to disappear.

**To Reproduce**
Rename a show with a bad renamer... Ie if your directory is TheNewShow (1954) and the rename uses TheNewShow..

**Expected behavior**
I would like the option to add the ShowYear to the rename,  I can't believe it's not there and I am hoping I just can't figure out the correct symbol..   There is airdate but that is not the shows date.. 

**Screenshots**

**Medusa (please complete the following information):**

Medusa Info: | Branch:                 master                  Commit:                 20915b56d64dc7522e484f326d9cfe7449a73193                  Version:                 0.4.1                  Database:                 44.16
-- | --
Python Version: | 3.7.3 (v3.7.3:ef4ec6ed12, Mar 25 2019, 22:22:05) [MSC v.1916 64 bit (AMD64)]
SSL Version: | OpenSSL 1.1.0j  20 Nov 2018
OS: | Windows-7-6.1.7601-SP1
Locale: | en_US.cp1252

**Debug logs (at least 50 lines):**
General > Advanced Settings > Enable debug
<details>

```
<-- Please replace this whole line with your debug logs -->
```
</details>

**Additional context**

**Comment by medariox** [comment]
This should already be the case. What is your naming pattern? Where are the debug logs and screenshots?

**Comment by nydave69** [comment]
Season %S/%SN (S%0SE%0E) (%Y-%0M-%0D) (%SQN) %EN

The log doesn't show anything...
The rename is doing what it's told..  but with out the year it tries to put it into the wrong place...

![image](https://user-images.githubusercontent.com/14140816/91910415-fcd2e180-ec7c-11ea-805c-d0bbf562984e.png)

the files need to go into the file structure...
I need the ability to add the show year to the name..

![image](https://user-images.githubusercontent.com/14140816/91910576-44596d80-ec7d-11ea-8886-6055964cf0ed.png)

**Comment by medariox** [comment]
The show name is given by the indexer. Shows that don't have an unique name will have the year appended, otherwise they won't.

**Comment by nydave69** [comment]
Any suggestion on how to fix my problem?
Can I add the year to the show name somehow?
Could we get the ability to add the show year in the renamer.   preferred..   %SY

thanks for taking the time..
